resources/test/fixtures/B008_extended.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+
+import fastapi
+from fastapi import Query
+
+
+def this_is_okay_extended(db=fastapi.Depends(get_db)):
+    ...
+
+
+def this_is_okay_extended_second(data: List[str] = fastapi.Query(None)):
+    ...
+
+
+def this_is_not_okay_relative_import_not_listed(data: List[str] = Query(None)):
```

Comment:
I feel like this should work / be considered ok. What do you think? (It doesn’t have to be part of this PR, just wondering if it’s _desired_ that it’s not accepted.)

Reply:
(I want to refactor some of the import-from tracking anyway.)

Reply:
I copied this from the flake8-bugbear repo so the name is probably misleading 🤔

Reply:
`convert_exit_to_sys_exit.get_member_import_name_alias()` could for this.
I can look into it.